configure.ac

```diff
@@ -152,6 +154,7 @@ echo "
 
     man pages (xsltproc):                         $enable_man
     SELinux:                                      $have_selinux
+     ... >= 2.3:                                  $have_selinux_2_3
```

Comment:
Is this really important enough to have in the summary? If anyone cares enough about this to go looking for this information, they can find it in the detailed output:

```
checking for libselinux >= 2.3... yes
```

I'm always a little sceptical about introducing a summary at the end of the configure step, because it's working around Autoconf's wall-of-text by ... adding more text? If we are going to have a summary, then it should be quite aggressively curated to only include parameters that are genuinely interesting, otherwise we will find ourselves wanting to add a summary of the summary!